var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -74,6 +74,9 @@ class Hypre(AutotoolsPackage, CudaPackage, ROCmPackage):
     variant("fortran", default=True, description="Enables fortran bindings")
     variant("gptune", default=False, description="Add the GPTune hookup code")
     variant("umpire", default=False, description="Enable Umpire support")
+    variant("gpu-aware-mpi", default=False, description="Use GPU-aware MPI")
+    variant("rocblas", default=False, description="Use rocBLAS")
```

Comment:
While rocblas is a valid configure option, hypre is *not* explicitly using rocBLAS so this variant may be misleading to users.